custom_model_runner/datarobot_drum/drum/artifact_predictors/keras_predictor.py

```diff
@@ -47,12 +47,13 @@ def can_use_model(self, model):
         try:
             import keras
             from sklearn.pipeline import Pipeline
+            from tensorflow import keras as keras_tf
 
             if isinstance(model, Pipeline):
                 # check the final estimator in the pipeline is Keras
-                if isinstance(model[-1], keras.Model):
+                if isinstance(model[-1], (keras.Model, keras_tf.Model)):
```

Comment:
Is there any parent class they both depend on?

Reply:
No, looks like there is no.
As I understand, Keras is API standard which can use different backends.

Starting tensorflow 2 - tensorflow implements keras.
https://blog.tensorflow.org/2018/12/standardizing-on-keras-guidance.html

So technically, tensorflow >= 2 doesn't require keras installation.

Reply:
That depends on the version of keras.  In the latest version of standalone keras, they're the same class. In the version included with the public env, the only common parent is object

Reply:
I see, other backends has been discontinued: https://github.com/keras-team/keras/blob/master/README.md